### PR TITLE
tikv-ctl: dynamically modify rocksdb cache size

### PR DESCRIPTION
**Pull request by UncP**

### CODE REVIEW

File: src/server/debug.rs [hunk 1]
```diff
@@ -415,10 +415,24 @@ impl Debugger {
             let cf = vec[0];
             let config_name = vec[1];
             validate_db_and_cf(db, cf)?;
+
             let handle = box_try!(get_cf_handle(rocksdb, cf));
-            let mut opt = Vec::new();
-            opt.push((config_name, config_value));
-            box_try!(rocksdb.set_options_cf(handle, &opt));
+            // currently we can't modify block_cache_capacity via set_options_cf
+            if config_name == "block_cache_capacity" {
```
Comment (AndreMouche) [inline src/server/debug.rs]: Could we modify the function `set_options_cf ` and let it support modify `block_cache_capacity `?

Comment (UncP) [reply src/server/debug.rs]: Not really.

**Comment by UncP** [comment]
@zhangjinpeng1987

File: src/server/debug.rs (hunk 1 above)
Comment (zhangjinpeng1987) [inline src/server/debug.rs]: Should keep consistent with cfg.

File: src/server/debug.rs [hunk 2]
```diff
@@ -415,10 +415,24 @@ impl Debugger {
...
+                let opt = rocksdb.get_options_cf(handle);
+                let capacity = config_value.parse::<u64>();
```
Comment (zhangjinpeng1987) [inline src/server/debug.rs]: Can we support MB and GB here?

**Comment by zhangjinpeng1987** [comment]
@huachaohuang PTAL

File: src/server/debug.rs (hunk 2 above)
Comment (UncP) [reply src/server/debug.rs]: Sure, but this can only be used on `cache size` here, other cfgs is passed to rocksdb.

**Pull request merged**